LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Denis Kudriashov
+Copyright (c) 2017-2021 Denis Kudriashov
```

Comment:
It's really not me. I just moved the project to github with little changes.
 @Ducasse are you the real author? And do you know who?